src/components/upload/nz-basic-upload.component.html

```diff
@@ -0,0 +1,11 @@
+<span [ngClass]="_classMap" (click)="onClick($event)" role="button">
+<input
+    #input
+    [accept]="accept"
+    style="display:none"
+    type="file"
+    (change)="onChange($event)"
+    [multiple]="multiple"
+>
```

Comment:
curiously, why not use ngModel?

Reply:
Thanks for your advice,whereas,I'm wondering about how to use ngModel in this scene.

Reply:
I maintain only one file list (only in nz-upload.component.ts ).